internal/connect/command_offset_update.go

```diff
@@ -0,0 +1,142 @@
+package connect
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/retry"
+)
+
+func (c *offsetCommand) newUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "update <id>",
+		Short:             "Update a connector's offsets.",
+		Args:              cobra.ExactArgs(1),
+		RunE:              c.update,
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Update offsets for a connector in the current or specified Kafka cluster context.",
+				Code: "confluent connect offset update lcc-123456 --config-file config.json",
+			},
+			examples.Example{
+				Code: "confluent connect offset update lcc-123456 --config-file config.json --cluster lkc-123456",
+			},
+		),
+	}
+
+	jsonExample := "\n{\n  \"offsets\": [\n    {\n      \"partition\": {\n        \"kafka_partition\": 0,\n        \"kafka_topic\": \"topic_A\"\n      },\n      \"offset\": {\n        \"kafka_offset\": 1000\n      }\n    }\n  ]\n}"
```

Comment:
We try to avoid messy multi-line descriptions like this one. A better place for this might be in the example above.

Reply:
Done